docker/.env.example

```diff
@@ -7,8 +7,12 @@ DB_USERNAME=postgres
 DB_PASSWORD=postgres
 DB_DATABASE_NAME=immich
 
-# Optional Database settings:
-# DB_PORT=5432
+# Remember you need to URL encode your password.
+# When you set DB_URL it overrides ALL the other DB ENV variables.
+# They are currently still used by the docker database container.
+# They should NOT be commented out as the validation still checks for them even if not used.
+
+# DB_URL=postgres://postgres:postgres@immich_postgres:5432/immich?sslmode=disable
```

Comment:
I consider DB_URL an advanced setting, to prevent confusion IMO we don't need to document it in `.env.example`, just on the docs website.